app/models/katello/glue/candlepin/subscription.rb

```diff
@@ -0,0 +1,64 @@
+module Katello
+  module Glue::Candlepin::Subscription
+    def self.included(base)
+      base.send :include, InstanceMethods
+      base.send :extend, ClassMethods
+    end
+
+    module ClassMethods
+      def candlepin_data(cp_id)
+        Katello::Resources::Candlepin::Subscription.get(cp_id)
+      end
+
+      def get_for_owner(organization = self.organization.label)
+        Katello::Resources::Candlepin::Subscription.get_for_owner(organization)
+      end
+    end
+
+    module InstanceMethods
+      def backend_data
+        self.class.candlepin_data(self.cp_id)
+      end
+
+      def redhat?
```

Comment:
This method looks like it could be on the active record model as its not specific to the Candlepin layer.